dashboard/src/components/Header/Header.tsx

```diff
@@ -47,11 +46,9 @@ function Header() {
     <section>
       <div className="container">
         <header className="header header-7">
-          <div className="branding">
-            <NavLink to="/">
-              <img src={logo} alt="Kubeapps logo" className="kubeapps__logo" />
-            </NavLink>
-          </div>
+          <NavLink to="/">
+            <div className="kubeapps__logo" />
+          </NavLink>
```

Comment:
This is causing the link not to be accessible because it doesn't contain any text inside. If you want to proceed with this solution, I would add a hidden text so screen readers can read at least "Homepage".

This is a common implementation of a `.sr-only` CSS class:

```
.sr-only {
	height: 1px;
	margin: -1px;
	overflow: hidden;
	padding: 0;
	position: absolute;
	width: 1px;
	white-space: nowrap;
}
```

Then, you can use it inside:

```suggestion
          <NavLink to="/">
            <div className="kubeapps__logo">
              <span className="sr-only">Homepage</span>
            </div>
          </NavLink>
```